src/schemas/clinical-resolvers/approveClinicalSubmission.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2023 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import submissionAPI from '../../submission/submission-api';
+import { GlobalGqlContext } from '../../app';
+
+const approveClinicalSubmissionResolver = {
```

Comment:
We can simplify these resolvers; instead of an object with a property, you can just declare the function itself:

````
const approveClinicalSubmission = async (...) => {...};
export default approveClinicalSubmission;
````

Then resolvers/mutations become:
````
Mutation: {
    approveClinicalSubmission,
    ...
    }
````
It's just fewer characters all around, easier to work with

Reply:
Example: https://github.com/icgc-argo/argo-clinical/pull/1060/files#diff-ec7c4660eea55c0aee57e6bc9f036f03444c350b145ad7086ba530baa7dd4d32

Reply:
Refactored the functions. Thanks for pointing out.